internal/osbuildexecutor/osbuild-executor.go

```diff
@@ -0,0 +1,11 @@
+package osbuildexecutor
```

Comment:
Commit title calls this osbuild**er**executor. I'd comment on the commit message, but you can't :)